types/tx.go

```diff
@@ -41,3 +45,19 @@ type TxProof struct {
 	Data     Tx               `json:"data"`
 	Proof    merkle.Proof     `json:"proof"`
 }
+
+func (txs Txs) ToTxsWithISRs(intermediateStateRoots IntermediateStateRoots) ([]pb.TxWithISRs, error) {
+	expectedLength := len(txs) + 2
```

Comment:
[nit] 
```suggestion
	expectedISRListLength := len(txs) + 2
```

Reply:
added

Reply:
Yes, it was called `expectedISRLength` but changed it to `expectedISRListLength`